<?xml version="1.0"?>
<launch>
	
	<node name="rosplan_interface_goto_waypoint" pkg="my_rosplan_interface" type="myaction" respawn="false" output="screen">
		<param name="knowledge_base" value="rosplan_knowledge_base" />
		<param name="pddl_action_name" value="goto_waypoint" />
		<param name="action_dispatch_topic" value="/rosplan_plan_dispatcher/action_dispatch" />
		<param name="action_feedback_topic" value="/rosplan_plan_dispatcher/action_feedback" />
	</node>

	<node name="go_to_point" pkg="motion_plan" type="go_to_point_action.py" respawn="false"/>
</launch>
